python/test/vmafexec_test.py

```diff
@@ -946,66 +946,27 @@ def test_run_vmafexec_runner_akiyo_multiply_no_enhn_gain_model_and_cmd_options(s
             None, fifo_mode=True,
             delete_workdir=True,
             result_store=None,
-            optional_dict={'disable_clip_score': True, 'model_filepath': VmafConfig.model_path("vmaf_v0.6.1neg.json"),
-                           'adm_enhn_gain_limit': 1.2}
+            optional_dict={'no_prediction': True, 'disable_clip_score': True, 'model_filepath': VmafConfig.model_path("vmaf_v0.6.1neg.json"),
+                           'adm_enhn_gain_limit': 1.2, 'vif_enhn_gain_limit': 1.0}
```

Comment:
Any reason why `vif_enhn_gain_limit` needs to be specified again? It is already been specified in vmaf_v0.6.1neg.json.

And why we cannot run prediction?